upkie/cpp/actuation/tests/BulletInterfaceTest.cpp

```diff
@@ -253,6 +253,33 @@ TEST_F(BulletInterfaceTest, MassRandomizationIsDifferent) {
   interface_->inertia_randomization_ = 0.0;
 }
 
+TEST_F(BulletInterfaceTest, EnvironmentCollision) {
+  interface_->reset(Dictionary{});
+  interface_->cycle([](const moteus::Output& output) {});
+  ASSERT_EQ(0, interface_->environment_collision());
+  std::string error;
+  std::unique_ptr<Runfiles> runfiles(Runfiles::CreateForTest(&error));
+  ASSERT_NE(runfiles, nullptr);
+  BulletInterface::Parameters params;
+  params.dt = dt_;
+  params.floor = false;   // wheels roll freely during testing
+  params.gravity = true;  // default, just a reminder
+  bullet::JointProperties left_wheel_props;
+  left_wheel_props.friction = kLeftWheelFriction;
+  params.joint_properties.try_emplace("left_wheel", left_wheel_props);
+  params.robot_urdf_path =
+      runfiles->Rlocation("upkie_description/urdf/upkie.urdf");
+  std::vector<std::string> env_urdf_paths;
+  env_urdf_paths.push_back(
+      runfiles->Rlocation("upkie_description/urdf/upkie.urdf"));
+  params.env_urdf_paths = env_urdf_paths;
+  interface_ = std::make_unique<BulletInterface>(params);
+  interface_->reset(Dictionary{});
+  interface_->cycle([](const moteus::Output& output) {});
+  ASSERT_EQ(1, interface_->environment_collision());
+  SetUp();
```

Comment:
Why call `SetUp` again here? This shouldn't be needed as it should be called before each test in the fixture (because we do a `TEST_F` rather than a `TEST`).